### ISSUE TITLE
should add moment to peerDependencies

### ISSUE THREAD

**Issue by GitNiko** [ant-design/ant-design#13352]
- [x] I have searched the [issues](https://github.com/ant-design/ant-design/issues) of this repository and believe that this is not a duplicate.

### Version
3.10.9

### Environment
all

### Reproduction link
[https://codepen.io/GitNiko/pen/gQQrKL](https://codepen.io/GitNiko/pen/gQQrKL)

### Steps to reproduce
dist/antd.min.js did not contain moment's source.
```html
<script
      src="https://unpkg.com/react@16.0.0/umd/react.production.min.js"
      async=""
    ></script>
    <script
      src="https://unpkg.com/react-dom@16.0.0/umd/react-dom.production.min.js"
      async=""></script>
    <!-- <script
      src="https://unpkg.com/moment@2.22.2/min/moment.min.js"
      async=""></script> -->
    <script
      src="https://unpkg.com/antd@3.0.1/dist/antd.min.js"
      async=""
    ></script>
```
it can't be runing without moment, so we should note in peerDependencies specially

### What is expected?
running

### What is actually happening?
Uncaught TypeError: ca(...) is not a function
    at Object.<anonymous> (Panel.js:193)
    at t (bootstrap e10f201ac5c7625931e6:19)
    at Object.<anonymous> (index.js:28)
    at t (bootstrap e10f201ac5c7625931e6:19)
    at Object.<anonymous> (antd.min.js:85443)
    at t (bootstrap e10f201ac5c7625931e6:19)
    at bootstrap e10f201ac5c7625931e6:62
    at bootstrap e10f201ac5c7625931e6:62
    at universalModuleDefinition:9
    at universalModuleDefinition:9

**Comment by ztplz** [comment]
![image](https://user-images.githubusercontent.com/17755202/49210634-93afa180-f3f8-11e8-90c2-f7014aed7653.png)
Why you comment?

**Comment by GitNiko** [comment]
@ztplz because peerDependencies just declare `react` and `react-dom`, it means `dist` contains anything you need except `react` and `react-dom`

**Comment by ztplz** [comment]
`moment` is so heavy, we should make `antd.min.js` as light as possible. It  is  better to install `moment` by user

**Comment by GitNiko** [comment]
Yes, that's right. But at least it should be clearly stated in the package.json. Not only just declare in `dependencies` but also declare in `peerDependencies`.
I am developing a tool that will auto generate the `<script src='unpkg/source'/>` base on the ` peerDependencies` of `package.json`.
so please add it to `peerDependencies`.

**Comment by DylanVann** [comment]
This was a misunderstanding.

Putting something in `peerDependencies` is explicitly saying "this should be installed by the user".

Anything in Webpack `externals` should also be in `peerDependencies`.

Putting something in `peerDependencies` will not include it in the build.
